finetuning/livecell/evaluation/evaluate_amg.py

```diff
@@ -21,28 +24,33 @@ def run_amg(name, model_type, checkpoint):
     return prediction_folder
 
 
-def eval_amg(name, prediction_folder):
+def eval_amg(name, prediction_folder, experiment_folder):
     print("Evaluating", prediction_folder)
     pred_paths, gt_paths = get_pred_and_gt_paths(prediction_folder)
-    save_path = os.path.join(get_experiment_folder(name), "results", "amg.csv")
+
+    if experiment_folder is None:
+        experiment_folder = get_experiment_folder(name)
+
+    save_path = os.path.join(experiment_folder, "results", "amg.csv")
     res = run_evaluation(gt_paths, pred_paths, save_path=save_path)
     print(res)
 
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument("-n", "--name", required=True)
+    parser.add_argument("-n", "--name", default=None)
```

Comment:
I would avoid creating two different arguments `name` and `experiment_folder`. If you think it's better to pass the experiment_folder then let's remove the `name` argument and make the experiment folder mandatory.

Reply:
Done! Thanks for the suggestion!